platform/flight.go

```diff
@@ -41,6 +51,32 @@ func NewBaseFlight(opts *Options, platform Name, ctPlatform string) (*BaseFlight
 	return NewBaseFlightWithDialer(opts, platform, ctPlatform, network.NewRetryDialer())
 }
 
+func GetEndianness() Endian {
```

Comment:
https://stackoverflow.com/questions/51332658/any-better-way-to-check-endianness-in-go
One of those seems better to me than shelling out to `lscpu` and parsing it...

Reply:
Also, rather than passing this value down the call stack, I think it'd be better to just determine it as a package `init()` function and export it.

Reply:
> https://stackoverflow.com/questions/51332658/any-better-way-to-check-endianness-in-go
> One of those seems better to me than shelling out to `lscpu` and parsing it...

I did try it yesterday but the first solution didn't work, so I just used `lscpu` and didn't look beyond. The solution from the tensor flow library seems to work. Will use that.

Reply:
> Also, rather than passing this value down the call stack, I think it'd be better to just determine it as a package `init()` function and export it.

Makes sense. Will do.